### PR TITLE
DTSSTCI-59 audit unit tests

### PR DESCRIPTION
**Pull request by AbdullahShahHMCTS**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-59

### Change description ###
Update coverage of tests
Correct tests where needed
Improve the assertion quality/coverage

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/app/form/Form.test.ts [hunk 1]
```diff
@@ -223,17 +223,17 @@ describe('Form', () => {
...
       fields: {
         customQuestion: {
           label: 'custom',
           type: 'text',
         },
       },
-    });*/
-    expect(1).toEqual(1);
+    });
+    //expect(1).toEqual(1);
```
Comment (JakeCG) [inline src/main/app/form/Form.test.ts:237]: Does this need deleting?

Comment (AbdullahShahHMCTS) [reply src/main/app/form/Form.test.ts:237]: yes, missed it as part of my testing - I'll clean it up

File: src/main/steps/privacy-policy/get.test.ts [hunk 2]
```diff
@@ -11,5 +12,10 @@ describe('PrivacyPolicyGetController', () => {
     const res = mockResponse();
     req.session.lang = 'en';
```
Comment (tomxelliott) [inline src/main/steps/privacy-policy/get.test.ts:13]: Why don't we create a test for cy as well?

File: src/main/steps/terms-and-conditions/get.test.ts [hunk 3]
```diff
@@ -10,5 +11,10 @@ describe('TermsAndConditionsGetController', () => {
     const req = mockRequest();
     const res = mockResponse();
     await controller.get(req, res);
+
+    expect(res.render).toHaveBeenCalledWith(
+      expect.stringContaining(__dirname + '/template'),
+      expect.objectContaining(en)
```
Comment (tomxelliott) [inline src/main/steps/terms-and-conditions/get.test.ts:17]: Why don't we create a test for cy as well?

File: src/main/steps/timed-out/get.test.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  const controller = new TimedOutGetController();
+
+  test('Should render the time out page', async () => {
+    const req = mockRequest();
+    const res = mockResponse();
+    await controller.get(req, res);
+
+    expect(res.render).toHaveBeenCalledWith(
+      expect.stringContaining(__dirname + '/template'),
+      expect.objectContaining(en)
```
Comment (tomxelliott) [inline src/main/steps/timed-out/get.test.ts:17]: Why don't we create a test for cy as well?

File: src/main/steps/edge-case/upload-supporting-documents/uploadDocPostController.test.ts [hunk 5]
```diff
@@ -276,6 +277,7 @@ describe('checking for the redirect of post document upload', () => {
       }];
```
Comment (tomxelliott) [inline src/main/steps/edge-case/upload-supporting-documents/uploadDocPostController.test.ts:277]: What is happening with this commented out block? If it is not in use, remove it

File: src/main/app/case/CaseApi.ts [hunk 6]
```diff
@@ -43,11 +43,11 @@ export class CaseApi {
    * @param caseId
    * @returns
    */
-  public async getCaseById(): Promise<CaseWithId> {
+  public async getId(): Promise<CaseWithId> {
     return new Promise(() => {
       null;
     });
-  }
+  } //TODO - Remove?
```
Comment (tomxelliott) [inline src/main/app/case/CaseApi.ts:50]: Move to to line 46 so it is clear which method you are talking about

File: src/main/app/form/Form.test.ts (hunk 1 above)
Comment (AbdullahShahHMCTS) [reply src/main/app/form/Form.test.ts]: Reverting this file back to master now, thanks

**Review by oleon23 — APPROVED** [review]

**Review by tomxelliott — APPROVED** [review]
